powermon/protocols/__init__.py

```diff
@@ -11,6 +11,7 @@ class ResponseType(Enum):
     ACK = auto()
     MULTIVALUED = auto()
     INDEXED = auto()
+    POSITIONAL = auto()
```

Comment:
responsetypes are a mess atm. trying to remove a bunch of old logic in abstractprotocol, but its wip